### PR TITLE
Add way to see only the errors logs

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/channels/views.py [hunk 1]
```diff
@@ -1481,13 +1481,19 @@ def derive_queryset(self, **kwargs):
...
+                if self.request.GET.get("errors"):
+                    logs = logs.filter(is_error=True)
+
+                events = logs.order_by("-created_on").select_related(
+                    "msg", "msg__contact", "msg__contact_urn", "channel", "channel__org"
                 )
-                patch_queryset_count(events, channel.get_non_ivr_log_count)
+
+                if self.request.GET.get("errors"):
+                    patch_queryset_count(events, channel.get_error_log_count)
```
Comment (rowanseymour) [inline temba/channels/views.py:1494]: Wondering about `.exclude(msg=None)` - we seem to include those in this count but not in the view.. and not in the index you've made for the view.. do we surface those errors elsewhere? Should we have this view include all errors including those without messages? 

(example of channel log without message is error from trying to fetch JioChat tokens)

Comment (norkans7) [reply temba/channels/views.py:1494]: Yeah, we can include all errors in the view as well

File: pyproject.toml [hunk 2]
```diff
@@ -58,6 +58,7 @@ python-intercom = "^3.1.0"
 packaging = "^20.9"
 requests-toolbelt = "^0.9.1"
 chardet = "^4.0.0"
+botocore = "^1.21.15"
```
Comment (rowanseymour) [inline pyproject.toml:61]: did you mean to update dependencies ?

Comment (norkans7) [reply pyproject.toml:61]: the other was failing to be installed

Comment (rowanseymour) [reply pyproject.toml:61]: We depend on `boto3` which in turn includes `botocore`.. I just tried and the current `boto3` installs fine for me. So if we did need to update it we just update `boto3` but I'm not sure this needs to be in this PR...

File: temba/channels/views.py (hunk 1 above)
Comment (rowanseymour) [reply temba/channels/views.py:1494]: have you checked the views are setup to support channel logs without messages?

File: pyproject.toml (hunk 2 above)
Comment (norkans7) [reply pyproject.toml:61]: I think it is an issue on python 3.9

```
Command ['/Users/norbert/nyaruka/rapidpro/.venv/bin/pip', 'install', '--no-deps', '-U', 'file:///Users/norbert/Library/Caches/pypoetry/artifacts/a9/ba/80/d188edc57d7fd21a4bc5d8c7c5523eab3fc56af72308d79a8c7fdf9e51/botocore-1.21.12-py3-none-any.whl'] errored with the following return code 2, and output: 
  Processing /Users/norbert/Library/Caches/pypoetry/artifacts/a9/ba/80/d188edc57d7fd21a4bc5d8c7c5523eab3fc56af72308d79a8c7fdf9e51/botocore-1.21.12-py3-none-any.whl
  ERROR: Exception:
  Traceback (most recent call last):
    File "/Users/norbert/nyaruka/rapidpro/.venv/lib/python3.9/site-packages/pip/_internal/cli/base_command.py", line 173, in _main
      status = self.run(options, args)
    File "/Users/norbert/nyaruka/rapidpro/.venv/lib/python3.9/site-packages/pip/_internal/cli/req_command.py", line 203, in wrapper
      return func(self, options, args)
    File "/Users/norbert/nyaruka/rapidpro/.venv/lib/python3.9/site-packages/pip/_internal/commands/install.py", line 315, in run
      requirement_set = resolver.resolve(
    File "/Users/norbert/nyaruka/rapidpro/.venv/lib/python3.9/site-packages/pip/_internal/resolution/resolvelib/resolver.py", line 75, in resolve
      collected = self.factory.collect_root_requirements(root_reqs)
    File "/Users/norbert/nyaruka/rapidpro/.venv/lib/python3.9/site-packages/pip/_internal/resolution/resolvelib/factory.py", line 471, in collect_root_requirements
      req = self._make_requirement_from_install_req(
    File "/Users/norbert/nyaruka/rapidpro/.venv/lib/python3.9/site-packages/pip/_internal/resolution/resolvelib/factory.py", line 433, in _make_requirement_from_install_req
      cand = self._make_candidate_from_link(
    File "/Users/norbert/nyaruka/rapidpro/.venv/lib/python3.9/site-packages/pip/_internal/resolution/resolvelib/factory.py", line 204, in _make_candidate_from_link
      self._link_candidate_cache[link] = LinkCandidate(
    File "/Users/norbert/nyaruka/rapidpro/.venv/lib/python3.9/site-packages/pip/_internal/resolution/resolvelib/candidates.py", line 295, in __init__
      super().__init__(
    File "/Users/norbert/nyaruka/rapidpro/.venv/lib/python3.9/site-packages/pip/_internal/resolution/resolvelib/candidates.py", line 156, in __init__
      self.dist = self._prepare()
    File "/Users/norbert/nyaruka/rapidpro/.venv/lib/python3.9/site-packages/pip/_internal/resolution/resolvelib/candidates.py", line 227, in _prepare
      dist = self._prepare_distribution()
    File "/Users/norbert/nyaruka/rapidpro/.venv/lib/python3.9/site-packages/pip/_internal/resolution/resolvelib/candidates.py", line 305, in _prepare_distribution
      return self._factory.preparer.prepare_linked_requirement(
    File "/Users/norbert/nyaruka/rapidpro/.venv/lib/python3.9/site-packages/pip/_internal/operations/prepare.py", line 508, in prepare_linked_requirement
      return self._prepare_linked_requirement(req, parallel_builds)
    File "/Users/norbert/nyaruka/rapidpro/.venv/lib/python3.9/site-packages/pip/_internal/operations/prepare.py", line 570, in _prepare_linked_requirement
      dist = _get_prepared_distribution(
    File "/Users/norbert/nyaruka/rapidpro/.venv/lib/python3.9/site-packages/pip/_internal/operations/prepare.py", line 61, in _get_prepared_distribution
      return abstract_dist.get_pkg_resources_distribution()
    File "/Users/norbert/nyaruka/rapidpro/.venv/lib/python3.9/site-packages/pip/_internal/distributions/wheel.py", line 26, in get_pkg_resources_distribution
      with ZipFile(self.req.local_file_path, allowZip64=True) as z:
    File "/usr/local/Cellar/python@3.9/3.9.2_2/Frameworks/Python.framework/Versions/3.9/lib/python3.9/zipfile.py", line 1257, in __init__
      self._RealGetContents()
    File "/usr/local/Cellar/python@3.9/3.9.2_2/Frameworks/Python.framework/Versions/3.9/lib/python3.9/zipfile.py", line 1324, in _RealGetContents
      raise BadZipFile("File is not a zip file")
  zipfile.BadZipFile: File is not a zip file
  

  at /usr/local/lib/python3.9/site-packages/poetry/utils/env.py:1101 in _run
      1097│                 output = subprocess.check_output(
      1098│                     cmd, stderr=subprocess.STDOUT, **kwargs
      1099│                 )
      1100│         except CalledProcessError as e:
    → 1101│             raise EnvCommandError(e, input=input_)
      1102│ 
      1103│         return decode(output)
      1104│ 
      1105│     def execute(self, bin, *args, **kwargs):

```

File: temba/channels/views.py (hunk 1 above)
Comment (norkans7) [reply temba/channels/views.py:1494]: Now that is handled

**Review by rowanseymour — APPROVED** [review]

**Pull request merged**